dist/select/ds6/select.css

```diff
@@ -26,8 +27,10 @@
   background-color: #fff;
   border: 1px solid #c7c7c7;
   border-radius: 0;
+  font-family: "Market Sans", Arial, sans-serif;
```

Comment:
@seangates @RajaRamu @senthilp  Will this interfere with how the ebay-font loader works? Will setting Market Sans explicitly in our CSS cause issues? I know we have avoided setting it explicitly in our global module, so just want to make sure here.

I also have another case where we need to set it explicitly for `button.fake-menu__item`.

Reply:
Did you try `inherit` for the button case? I briefly tried it when I filed that issue, and it seemed to work.

Reply:
@ianmcburnie As @yomed mentioned we should first try with `inherit`. Setting Market Sans explicitly will cause FOUT/FOIT in some browsers, as `font-display` [support](https://caniuse.com/#search=font-display) is still not widely available.

Reply:
Looks like resolved now 👍

Reply:
It's resolved for the other issue in menu, but not here for select.

Reply:
Correct. For one reason or another, the `<select>` doesn't inherit it from anywhere and it needs to be explicitly set.

Reply:
The other solution would be to use a more specific selector, just for `font-family`
```css
.font-marketsans .select > select {
    font-family: "Market Sans", Arial, sans-serif;
}
```
[`ebay-font`](https://github.com/senthilp/ebay-font/blob/master/font/marketsans/template.marko#L11) adds that class `font-marketsans` only when Market Sans is available and it should get the best of both worlds.

Reply:
@senthilp I was not aware of the `.font-marketsans` class. I will go ahead and add that selector.